cmd/queue/main.go

```diff
@@ -388,6 +388,7 @@ func main() {
 		composedHandler = pushRequestMetricHandler(composedHandler, requestCountM, responseTimeInMsecM, env)
 	}
 	composedHandler = tracing.HTTPSpanMiddleware(composedHandler)
+	composedHandler = network.NewProbeHandler(composedHandler)
```

Comment:
Is this one going to replace the probe requests check part in `func handler` in future?

And same for the handlers order. I don't know why currently metric handler is not the outermost one but it should be so that the latency is the time added by all handlers.

Reply:
Queue-Proxy probing is different than this probing, so I don't think they'll ever be merged.

Regarding the ordering, you can argue that tracing also needs to be the outer most.
But all these handlers (metrics, tracing, probing) are not all doing sub-millisecond work, plus the metrics handler already filters out probing requests, so it doesn't matter here.